packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -65,9 +66,11 @@ abstract class ScrollView extends StatelessWidget {
   Widget build(BuildContext context) {
     List<Widget> slivers = buildSlivers(context);
     AxisDirection axisDirection = getDirection(context);
-    return new Scrollable(
+
+    ScrollController primaryScrollController = PrimaryScrollController.of(context);
```

Comment:
Only call this if `primary` is true. It sets up a relationship with the other widget that isn't cheap.